example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -144,25 +143,6 @@ describe('The S3 Ingest Granules workflow', () => {
     expectedPayload = loadFileWithUpdatedGranuleIdPathAndCollection(templatedOutputPayloadFilename, granuleId, testDataFolder, newCollectionId);
     expectedPayload.granules[0].dataType += testSuffix;
 
-    // pre-stage destination files for MoveGranules
-    const preStageFiles = expectedPayload.granules[0].files.map((file) => {
```

Comment:
Did this just get moved because I'm not seeing it? This was added to test that moveGranules will overwrite duplicate files by default and add a `duplicate_found` flag on each duplicate file that was encountered.

Reply:
Ah, is the idea that there's no need to pre-stage the destination files because the re-ingest execution will handle testing this case?

Reply:
Yeah, I try not to add more tests to the integration-test.  `reingest` will also test `replace` functionality.

Reply:
Good thinking!